### PR TITLE
Reduce duration of wait, recreate client on each request

### PR DESCRIPTION
**Pull request by aug24**

### CODE REVIEW

File: src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala [hunk 1]
```diff
@@ -19,25 +19,26 @@ object HTTP extends Logging {
...
         logger.info(s"Status of GET request was ${result.status}")
         result.status match {
           case 200 => Right(result.body)
           case _ => Left(RCSError(result.status + ": " + result.body))
         }
-      }, 10.minutes)
+      }, 2.minutes)
     } catch {
       case e: Throwable =>
         wsClient.close()
```
Comment (sihil) [inline src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala]: If you close this in the `finally` it doesn't need to be here. Not sure what happens if it gets called twice.

**Review by sihil — COMMENTED** [review]
Looking good, not sure about the failure case when closing the client.

File: src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala (hunk 1 above)
Comment (aug24) [reply src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala]: Damn, I had removed it from here and accidentally put it back.

File: src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala [hunk 2]
```diff
@@ -19,25 +19,25 @@ object HTTP extends Logging {
...
+    val wsClient = StandaloneAhcWSClient()
     try {
       Await.result(wsClient.url(rcsUrl).withQueryStringParameters(("lastid", lastid.toString), ("subscribername", subscriberName)).get().map { result =>
         logger.info(s"Status of GET request was ${result.status}")
         result.status match {
           case 200 => Right(result.body)
           case _ => Left(RCSError(result.status + ": " + result.body))
         }
-      }, 10.minutes)
+      }, 2.minutes)
```
Comment (akash1810) [inline src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala:33]: does this need to relate to the [lambda timeout](https://github.com/guardian/rcs-poller-lambda/blob/master/cfn.yaml#L149)?

Comment (aug24) [reply src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala:33]: It leads to a longer delay before detection of a failure.  I sincerely hope that we are not doing a call to check connectivity which takes the server ten minutes to process..?

Comment (akash1810) [reply src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala:33]: We're talking to a web server hosted within an Oracle database... who knows how long it'll take...

**Review by akash1810 — APPROVED** [review]

**Pull request merged**